packages/fxa-payments-server/src/routes/Product/AcceptedCards/index.tsx

```diff
@@ -5,15 +5,13 @@ import mastercardLogo from '../../../images/mastercard.svg';
 import discoverLogo from '../../../images/discover.svg';
 import amexLogo from '../../../images/amex.svg';
 
-import './index.scss';
-
 export const AcceptedCards = () => {
   return (
-    <div className="accepted-cards">
-      <img src={visaLogo} alt="visa" />
-      <img src={mastercardLogo} alt="mastercard" />
-      <img src={discoverLogo} alt="discover" />
-      <img src={amexLogo} alt="american express" />
+    <div className="flex justify-center mb-10">
```

Comment:
Could we replace the individual paddings below with `gap-2`?

I also noticed that `mb-10` seems to be much bigger than what was there before.